server/server.go

```diff
@@ -166,17 +216,24 @@ func (server *HTTPServer) Initialize(address string) http.Handler {
 		router.Use(server.Authentication)
 	}
 
+	apiPrefix := server.Config.APIPrefix
+
 	// common REST API endpoints
-	router.HandleFunc(server.Config.APIPrefix, server.mainEndpoint).Methods("GET")
-	router.HandleFunc(server.Config.APIPrefix+"organizations", server.listOfOrganizations).Methods("GET")
-	router.HandleFunc(server.Config.APIPrefix+"organizations/{organization}/clusters", server.listOfClustersForOrganization).Methods("GET")
-	router.HandleFunc(server.Config.APIPrefix+"report/{organization}/{cluster}", server.readReportForCluster).Methods("GET")
+	router.HandleFunc(apiPrefix, server.mainEndpoint).Methods("GET")
+	router.HandleFunc(apiPrefix+"organizations", server.listOfOrganizations).Methods("GET")
+	router.HandleFunc(apiPrefix+"report/{organization}/{cluster}", server.readReportForCluster).Methods("GET")
+	router.HandleFunc(apiPrefix+"rule/{cluster}/{rule_id}/like", server.likeRule).Methods("PUT")
```

Comment:
Please follow API designing guidelines https://platform-docs.cloud.paas.psi.redhat.com/rest/urls.html
I should look like this:
`clusters/{cluster_id}/rules/{rule_id}/like`

Reply:
fixed